web/libs/editor/src/stores/Annotation/Annotation.js

```diff
@@ -80,14 +81,21 @@ const hotkeys = Hotkey("Annotations", "Annotations");
  * @param value {Object} object to fix
  * @returns {Object} new object without value fields
  */
-function omitValueFields(value) {
+const omitValueFields = ff.isActive(ff.FF_CUSTOM_TAGS) ? (value) => {
+  // @todo describe that we only omit `text` from TextArea
```

Comment:
Is this true? The original code looks like it is possibly omitting far more 🤯

Reply:
Yeah, we needed it only because of this confusion. I added some info into commit message (https://github.com/HumanSignal/label-studio/pull/8108/commits/60ca64c07c31f4f05d3a8b5d299b95d1f96f2996), but will write a comment here as well.